packages/x-domain/test/ovmcontext.spec.ts

```diff
@@ -0,0 +1,111 @@
+import assert = require('assert')
```

Comment:
I would prefer that we use `expect`, since that's the pattern we're using in our tests elsewhere. It should be available via `import { expect } from './setup'`